Source/Shared/BPUtils.h

```diff
@@ -69,11 +69,10 @@ typedef NS_ENUM(int, BPKind) {
 + (void)printInfo:(BPKind)kind withString:(NSString *)fmt, ...;
 
 /*!
- @discussion print a message to stderr.
- @param kind one of the levels in BPKind
- @param fmt a format string (a la printf), followed by the var args.
+ @discussion get an NSError *
+ @param fmt a format string (a la printf), followed by var args.
  */
-+ (void)printError:(BPKind)kind withString:(NSString *)fmt, ...;
++ (NSError *)BPError:(NSString *)fmt, ... ;
```

Comment:
😂 tbh, I don't know why, but that's the convention I have seen many times. Let me find a sample and try to get an explanation